ArduCopter/Copter.h

```diff
@@ -125,6 +125,11 @@
 #include "afs_copter.h"
 #endif
 
+
+#if USE_WAYBACK == ENABLED
+#include "/AP_WayBack/AP_WayBack.h"
```

Comment:
yes but as I sayd I never build this library into real Copter code so all changes outside AP_Wayback folder are just "proof-of-concept", samples of usage and needs to be corrected